client/src/components/Import.jsx

```diff
@@ -1,29 +1,139 @@
-import React, { Component } from 'react';
 
-class App extends Component {
-  constructor(props) {
-    super(props);
-  }
+import { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import CardContainer from './card/CardContainer';
+import Popup from 'reactjs-popup';
+import './Import.css';
+
+function ImportFileUpload({ onImportLogs }) {
+  const showFile = async (e) => {
 
-  showFile = async (e) => {
     e.preventDefault();
     const reader = new FileReader();
     reader.onload = async (e) => {
       const text = e.target.result;
 
-      const importedData = JSON.parse(text);
-      console.log(importedData);
+
+      onImportLogs(text);
+      uploadInfoToggle();
+
     };
     reader.readAsText(e.target.files[0]);
   };
 
-  render = () => {
-    return (
-      <div>
-        <input type='file' onChange={(e) => this.showFile(e)} />
-      </div>
-    );
+
+  const [uploadInfo, setUploadInfo] = useState(false);
+
+  useEffect(() => {
+    if (!uploadInfo) return;
+
+    const uploadTimer = setTimeout(() => {
+      setUploadInfo(false);
+    }, 5000);
+
+    return () => {
+      clearTimeout(uploadTimer);
+    };
+  }, [uploadInfo]);
+
+  const uploadInfoToggle = () => {
+    setUploadInfo(true);
   };
+
+  return (
+    <CardContainer>
+      <Grid>
+        <div className='text'>Import your Logs</div>
+        <div className='link-import'>
+          <Popup
+            trigger={<FakeButton>Upload CSV</FakeButton>}

```

Comment:
Why'd you call the trigger FakeButton ? 😄

Reply:
Because a "real" Button is making troube with the Styles due to the Popup Plugin :)